tests/integration/test_charm.py

```diff
@@ -29,11 +29,11 @@ async def test_reachable(app: Application):
     """
     # Get the IP address which is in the status message
     ip_regex = r"[0-9]+(?:\.[0-9]+){3}"
-    ip_address_match = re.search(ip_regex, app.units[0].workload_status_message)
+    ip_address_match = re.findall(ip_regex, app.units[0].workload_status_message)
```

Comment:
Out of curiosity, why the switch from `search` to `findall`?

Reply:
because now the message has 2 IPs, and we want only the second one. Re.search() always returns the first ocurrence of the pattern.